src/app/public/modules/timepicker/timepicker.component.spec.ts

```diff
@@ -547,8 +547,7 @@ describe('Timepicker', () => {
       expect(getInput(fixture)).not.toHaveCssClass('ng-invalid');
     }));
 
-    // TODO: This should be fixed with issue https://github.com/blackbaud/skyux-datetime/issues/135
-    xit('should properly update model and input when required and undefined', fakeAsync(() => {
+    it('should properly update model and input when required and undefined', fakeAsync(() => {
```

Comment:
Unrelated change to this PR. Just figured out this was working as expected - just needed some more `ticks` in there.